loader_hub/web/readability_web/base.py

```diff
@@ -77,9 +87,16 @@ def load_data(self, url: str) -> List[Document]:
                 "siteName",
             ]}
 
+            article["textContent"] = self._normalize(article["textContent"])
```

Comment:
if self._normalize allowed to be None? if so we should add a check to see if it's none first